### PR TITLE
add a note about GPG/IRC for PMC members and update collaborator election process

### PR DESCRIPTION
**Pull request by bastelfreak**

### CODE REVIEW

File: share/governance.md [hunk 1]
```diff
@@ -61,7 +61,7 @@ Membership of the PMC is by election. [Condorcet](https://en.wikipedia.org/wiki/
 
 The PMC doesn't have specific roles or a chairperson.
 
-
+At the moment, IRC is our main communication channel. PMC members should be present in our `#voxpupuli-pmc` channel on freenode. Also we have some gp encrypted credentials and a few repositories require gpg-signed commits, so a PMC member also needs a gpg key.
```
Comment (natemccurdy) [inline share/governance.md]: `gp encrypted` -> `gpg encrypted` ?

Comment (bastelfreak) [reply share/governance.md]: Thanks. I've the strong feeling that my keyboard is dying :(

File: share/governance.md [hunk 2]
```diff
@@ -61,7 +61,7 @@ Membership of the PMC is by election. [Condorcet](https://en.wikipedia.org/wiki/
 
 The PMC doesn't have specific roles or a chairperson.
 
-
+At the moment, IRC is our main communication channel. PMC members should be present in our `#voxpupuli-pmc` channel on freenode. Also we have some gpp encrypted credentials and a few repositories require gpg-signed commits, so a PMC member also needs a gpg key.
```
Comment (natemccurdy) [inline share/governance.md]: > gpp encrypted

gpp?

And to nitpick a bit, might as well hyphenate this like you do with `gpg-signed commits`:
> gpg-encrypted credentials

Comment (bastelfreak) [reply share/governance.md]: ups, fixed it

**Comment by bastelfreak** [comment]
The stuff from the second commit isn't new. I just updated it to reflect the reality.

**Pull request merged**